packages/payment-streams/src/index.js

```diff
@@ -7,11 +7,13 @@ const erc20Abi = require('erc-20-abi')
 const pTap = require('p-tap').default
 
 const { findToken } = require('./token-list')
-const contracts = require('./contracts.json')
 const createExecutor = require('eth-exec-txs')
 const paymentStreamAbi = require('./abis/PaymentStream.json')
 const paymentStreamFactoryAbi = require('./abis/PaymentStreamFactory.json')
 
+const DEFAULT_ADDRESS = '0xf47853220BF59C682CCBd28e7cFF4D4E603a4Ca6' // Chain ID 1
```

Comment:
same here re: `Chain ID 1`

Reply:
Removed the default here: https://github.com/hemilabs/pure.finance/pull/2/commits/50613c94ab27cb3cc111f820f6ba587e14cf0d97